CNV#50738: Doc: Update downstream doc Delete Protection for VMs (#90981)
openshift/openshift-docs: modules/virt-enabling-disabling-vm-delete-protection-web.adoc

// Module included in the following assemblies:
//
// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-enabling-disabling-vm-delete-protection-web_{context}"]

> = Enabling or disabling virtual machine delete protection by using the web console

Reviewer: ```suggestion
= Enabling or disabling VM delete protection by using the web console
```

> To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the {product-title} web console. You can also disable delete protection for a VM.

Reviewer: ```suggestion
To prevent the inadvertent deletion of a virtual machine (VM), you can enable VM delete protection by using the {product-title} web console. You can also disable delete protection for a VM.
```

// Module included in the following assemblies:
//
// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-enabling-disabling-vm-delete-protection-web_{context}"]

= Enabling or disabling virtual machine delete protection by using the web console

To prevent the inadvertent deletion of a virtual machine, you can enable virtual machine delete protection by using the {product-title} web console.

.Prerequisites

* You have installed {VirtProductName} 4.19 or later.

> * You have installed the OpenShift CLI (`oc`).

Reviewer 2: In the case of web console, users do not need `oc`. IMHO, we can drop this prerequisite

// Module included in the following assemblies:
//
// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc

:_mod-docs-content-type: PROCEDURE
[id="virt-enabling-disabling-vm-delete-protection-web_{context}"]

= Enabling or disabling virtual machine delete protection by using the web console

> To prevent the inadvertent deletion of a virtual machine, you can enable virtual machine delete protection by using the {product-title} web console. Should you change your mind, you can easily disable delete protection for a VM.

Reviewer 2: ```suggestion
To prevent the inadvertent deletion of a VM, you can enable virtual machine delete protection by using the {product-title} web console. Should you change your mind, you can easily disable delete protection for a VM.
```

> To prevent the inadvertent deletion of a virtual machine (VM), you can enable virtual machine delete protection by using the {product-title} web console. Should you change your mind, you can easily disable delete protection for a VM.

Reviewer 3: Some of the same comments in the above module apply here, if you can check those

By default, VMs are not enabled with delete protection. You must set the option for each individual VM.

.Prerequisites

> * You have installed {VirtProductName} 4.19 or later.

Reviewer 3: Same about removing this

.Procedure

. From the {product-title} web console, choose your view:

>     * For a virtualization-focused view, select *Administrator* → *Virtualization* → *VirtualMachines*.

Reviewer 3: Same about not showing choices if you don't need to

    * For a general view, navigate to *Virtualization* → *VirtualMachines*.

. From the *VirtualMachines* list, select the VM whose delete protection you want to enable or disable.

. Click the *Configuration* tab.

> . In the *VirtualMachines details*, choose to enable or disable the protection as follows:

Reviewer 3: I don't love "as follows", but I don't know if there is actual guidance against it

* To enable the protection:
.. Set the *Deletion protection* switch to *On*.
.. Click *Enable* to confirm the protection.

 * To disable the protection:
.. Set the *Deletion protection* switch to *Off*.

> .. Click *Disable* to confirm the protection.

Reviewer 3: Instead of "confirm the protection" should this be something like "disable the protection" or "remove the protection"?
